### DISCUSSION TITLE
BetterEnderchest Support

### DISCUSSION THREAD

**Discussion by puqitv** [TechnicallyCoded/Inventory-Rollback-Plus#75] [Ideas]
Yo I think it would be great if this plugin could support the betterenderchest plugin. There could be some sort of setting in the config that says “use betterenderchest plugin: true” or something.

**Comment by TechnicallyCoded** [comment]
That would require saving data for every enderchest they own, every time they die, change world, join, leave.... It seems like a pretty computationally expensive task. I don't think this is the right plugin to integrate these backups into. Maybe contact the devs of BetterEnderChest to add time based backups.
